src/components/question-genre-screen/question-genre-screen.jsx

```diff
@@ -0,0 +1,88 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {GameType} from "../../const.js";
+class QuestionGenreScreen extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      answers: [false, false, false, false],
+    };
+  }
```

Comment:
если в конструкторе ничего кроме стейта не инициализируется, можно использовать сокращенную нотацию (https://www.robinwieruch.de/react-state-without-constructor)
```suggestion
  state = {
    answers: [false, false, false, false],
  };
```

Reply:
Но это требует установки доп пакета. На ваше усмотрение

Reply:
При использовании сокращенной нотации смущает потеря возможности доступа к пропсам (However, you don't have access to the props anymore.)
Я попробовал эту нотацию, но получаю ошибку eslint (Parsing error: Unexpected token = ). Как победить eslint и какой устанавливать доп пакет я не понял.

Reply:
разобрался. 
доступ к пропсам не пропал, 
доп пакеты потребовались: @babel/plugin-proposal-class-properties и babel-eslint.